src/actions/actions.ts

```diff
@@ -3537,20 +3537,55 @@ class ToggleCaseOperator extends BaseOperator {
 
   public async run(vimState: VimState, start: Position, end: Position): Promise<VimState> {
     const range = new vscode.Range(start, new Position(end.line, end.character + 1));
```

Comment:
Can you also change this to: `new vscode.Range(start, end.getRight());`.

Probably doesn't matter but I'd sleep better at night with the position checking.